### ISSUE TITLE
Add support for rootless jailbreak

### ISSUE THREAD

**Issue by Supposedly-Sam** [AwesomeIndustry/SIMLessPNRGateway#9]
Rootful jailbreak support is being phased out by palera1in so please support for rootless jailbreak

**Comment by Nickthegoatboy** [comment]
Yes, I'm trying to set this up on my rootless iPhone 7 now and not having any luck. 

Is it possible just to change the directory that you're looking for the phone number in?
